tests/integration/test_vulnerability_detector/test_windows/test_cpe_indexing.py

```diff
@@ -130,22 +130,22 @@ def get_configuration(request):
 
```

Comment:
This line is not necessary
```
wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
```
The function `restart_modulesd` already defined the FileMonitor object.